src/libcmd/installables.cc

```diff
@@ -161,15 +161,15 @@ SourceExprCommand::SourceExprCommand(bool supportReadOnlyMode)
 
     addFlag({
         .longName = "expr",
-        .description = "Interpret installables as attribute paths relative to the Nix expression *expr*.",
+        .description = "Interpret *installables* as attribute paths relative to the Nix expression *expr*.",
         .category = installablesCategory,
         .labels = {"expr"},
         .handler = {&expr}
     });
 
     addFlag({
         .longName = "derivation",
-        .description = "Operate on the [store derivation](../../glossary.md#gloss-store-derivation) rather than its outputs.",
+        .description = "If *installables* is a [store derivation](../../glossary.md#gloss-store-derivation), operate on that rather than its outputs.",
```

Comment:
```suggestion
        .description = "If *installables* is a [store derivation](@docroot@/glossary.md#gloss-store-derivation), operate on that rather than its outputs.",
```